index.js

```diff
@@ -48,29 +48,32 @@ app.get('/', (req, res) => {
 })
 
 app.get('/admin', auth, async (req, res) => {
-  const result = await query('SELECT * FROM company;')
+  const result = await dashboardHelper.query('SELECT * FROM company;')

```

Comment:
I think it would be neater if we pulled the `query` function in to a separate module that both this and the `dashboardHelper` could depend on. Ideally we would have something like:

```
const { query } = require('databaseClient')
```